float32 factor
---
int16 status